conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /hec-stubs              app.Routes
+->         /                       app.Routes
```

Comment:
Shouldn't we put hec-stubs or application name in here?

Reply:
we neec to be able to call our stub in place of the real things without changing the url structure in the calling code, so the stub needs to be able to listen out for calls as if they were the real thing. If we left the service name in there we'd have to include it in the calling code optionally which is slightly annoying